wsgi/app.py

```diff
@@ -0,0 +1,29 @@
+"""
```

Comment:
Suggestion: If we're moving the wsgi file, the typical Django pattern (these days at least) is to add it to the app package. e.g. `kitsune/wsgi.py`. It's not what all our sites do though... so either way.

Reply:
Since we also have `config.py` probably better to keep them both under `wsgi/`